enrollment-server/src/main/java/com/wultra/app/enrollmentserver/configuration/PowerAuthWebServiceConfiguration.java

```diff
@@ -41,8 +41,7 @@
 @Configuration
 @ConfigurationProperties("ext")
 @ComponentScan(basePackages = {
-        "com.wultra.security.powerauth",
-        "io.getlime.security.powerauth",
+        "com.wultra.security.powerauth"
```

Comment:
Nitpick: It may be simplified even into `@ComponentScan("com.wultra.security.powerauth")`

Reply:
It is not crucial (just „figure skating“ 😉), but the point was that `value` is an alias for `basePackages`, so it is not needed at all.
```java
@AliasFor("value")
String[] basePackages() default {};
```

Reply:
Ah, ok, sure.